spacifier.js

```diff
@@ -0,0 +1,10 @@
+function _spacify(str){ return str.split('').join(' ');}
```

Comment:
please be consistent in function declaration

```suggestion
const _spacify = str => str.split('').join(' ');
```